lib/internal/modules/cjs/loader.js

```diff
@@ -1400,10 +1400,11 @@ Module._extensions['.js'] = function(module, filename) {
     const pkg = packageJsonReader.readPackageScope(filename) || { __proto__: null };
     // Function require shouldn't be used in ES modules.
     if (pkg.data?.type === 'module') {
+      // This is an error path, because `require` of a `.js` file in a `"type": "module"` scope is not allowed.
```

Comment:
nit

```suggestion
      // This is an error path because `require` of a `.js` file in a `"type": "module"` scope is not allowed.
```

Reply:
Meh. I’ve already started CI, I don’t want to start over because of a comma 😄